### DISCUSSION TITLE
DT4ALL Android

### DISCUSSION THREAD

**Discussion by Alp1ne** [cedricp/ddt4all#947] [General]
Hi there, can I use this app to activate rear camera on a car after installing the loom, same as with the desktop DT4ALL? If so I would like to purchase a license please. Many thanks

**Comment by Furtif** [comment]
> Hi there, can I use this app to activate rear camera on a car after installing the loom, same as with the desktop DT4ALL? If so I would like to purchase a license please. Many thanks

not need licenses https://github.com/cedricp/ddt4all/releases/tag/v3.0.0
